fighthealthinsurance/management/commands/generate_blog_metadata.py

```
@@ -41,22 +41,34 @@ def handle(self, *args, **options):
                     try:
                         with open(path, "r", encoding="utf-8") as f:
                             content = f.read()
-                    except Exception as e:
+                    except OSError as e:
                         self.stderr.write(
                             self.style.ERROR(f"Failed to read {path}: {e}")
                         )
                         continue
+                    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)